RELEASE/scripts/autoscend/quests/level_05.ash

```diff
@@ -10,6 +10,13 @@ boolean L5_getEncryptionKey()
 	{
 		return false;
 	}
+
+	if (is_banished($phylum[goblin]) && get_property("screechCombats").to_int() > 0)
```

Comment:
there are only goblins in the outskirts, so if they are banished they will all come back, so adventuring is fine

unless you have a habitated monster, in which case you will only fight your habitated monster